conf/apt/sources.list.d/kano-jessie.list

```diff
@@ -0,0 +1,32 @@
+#
+# Kano OS Jessie Official Package Repository
+# Key:
+#   curl -L http://repo.kano.me/archive-stretch/repo.gpg.key | apt-key add -
+#
+
+deb http://repo.kano.me/archive-jessie/ release main
+deb http://dev.kano.me/archive-jessie/ devel main
```

Comment:
Let's create a test case where we make sure that when the dev server goes offline, the Updater doesn't report an error and exit.